docs/source/api/v2/asns_id.rst

```diff
@@ -23,6 +23,8 @@
 
 ``GET``
 =======
+.. deprecated:: ATCv4
```

Comment:
deprecation notice should go in the v1 documentation and then this GET documentation should be deleted